src/services/AppsApiService.js

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+import config from '../config';
+
+export default class AppsApiService {
+  static async get(authToken) {
+    const url = AppsApiService.#buildUrl();
+
+    return axios.get(url, {
+      headers: {
+        Authentication: authToken,
+      },
+    });
+  }
+
+  static async getById(authToken, { id }) {
```

Comment:
I think we don't need `async` keyword here and in other places

Reply:
> I think we don't need `async` keyword here and in other places

agree with Dima, its not good block stream